packages/sdk/src/types/Customers.ts

```diff
@@ -121,6 +123,95 @@ export interface CustomerActivitiesListResponse {
 	data: Record<string, any>[]
 }
 
+export interface CustomerRedeemablesListQueryParams {
+	limit?: number
+	order?: 'created_at' | '-created_at' | 'id' | '-id'
+	starting_after_id?: string
+	filters?: Record<string, any>
+}
+
+export interface CustomerRedeemablesListResponse {
+	object: 'list'
+	total: number
+	data_ref: 'data'
+	data: CustomerRedeemable[]
+	has_more: boolean
+	more_starting_after?: string
+}
+
+export interface CustomerRedeemable {
```

Comment:
If those types are going to be used only for this single endpoint, then we should follow CONTRIBUTING.md and the naming convention for 0-level types; in this case, it should be something like this:
- CustomerRedeemablesListCustomerRedeemableResponse
- CustomerRedeemablesListCustomerRedeemableContainerResponse
- CustomerRedeemablesListCustomerRedeemableContainerVoucherResponse